src/__test__/addNew.test.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import addNew from '../addNew.js';
+
+jest.mock('../addNew.js');
+
+describe('Testing addNew To Do task function', () => {
+  test('Check the number of elements added, should be 1', () => {
+    expect(addNew()[0]).toBe(1);
+  });
+
+  test('Check if the name equals First', () => {
+    expect(addNew()[1]).toBe('First');
+  });
+});
```

Comment:
- [ ] Let's also test:
- Function for editing the task description
- Function for updating an item's 'completed' status
- The 'clear all completed' function

Reply:
can you be more descriptive

Reply:
You wrote function  to test 'addNewtask'. Can you write some tests to check function for 'editing task', 'update task' and 'clear all completed task' ? Good luck!